recipe/meta.yaml

```
@@ -32,23 +37,43 @@ requirements:
     - typing_extensions
 
 test:
+  source_files:
+    - scripts
+    - python-package
+    - resources
+  files:
+    - pytest.ini  # [win or s390x or py>=313]  # to exclude polars test
   imports:
     - perpetual
   commands:
     - pip check
+    - python scripts/make_resources.py
+    - sh scripts/run-python-tests.sh  # [not win]
+    - scripts\run-python-tests.ps1  # [win]
   requires:
     - pip
+    - pytest
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - maturin
+    - pandas
+    - sklearn-compat
+    - seaborn
+    - pyarrow
```

```suggestion
test:
  source_files:
    - scripts/make_resources.py
    - python-package/tests
    - resources
  imports:
    - perpetual
  commands:
    - pip check
    - python scripts/make_resources.py
    - cd python-package
    - pytest -vv tests -k "not test_polars"  # [win or s390x or py>=313]
    - pytest -vv tests  # [not (win or s390x or py>=313)]
  requires:
    - pip
    - pytest
    - pandas
    - sklearn-compat
    # seaborn used in scripts/make_resources.py
    - seaborn
    # pa.array requires 'pyarrow' module to be installed
    # when using polars
    - pyarrow  # [not (win or s390x or py>=313)]
```

see internal discussion on https://anaconda.slack.com/archives/C03J3NQPS0Z/p1740775454077749?thread_ts=1740753701.018109&cid=C03J3NQPS0Z